ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkbox/v1/checkbox/_cq_dialog/.content.xml

```diff
@@ -48,24 +48,57 @@
                                             <placeholder
                                                     jcr:primaryType="nt:unstructured"
                                                     sling:hideResource="{Boolean}true"/>
-                                            <enumOptions jcr:primaryType="nt:unstructured"
-                                              granite:class="cmp-adaptiveform-checkbox__options"
-                                              sling:resourceType="granite/ui/components/coral/foundation/form/multifield"
-                                              fieldDescription="Provide the pair of values representing 'ON' or 'OFF' state"
-                                              fieldLabel="Options">
-                                                <field jcr:primaryType="nt:unstructured"
-                                                  sling:resourceType="granite/ui/components/coral/foundation/container"
-                                                  name="./enum">
-                                                    <items jcr:primaryType="nt:unstructured">
-                                                        <enum jcr:primaryType="nt:unstructured"
-                                                          sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
-                                                          granite:class="cmp-adaptiveform-base__enum"
-                                                          name="./enum"
-                                                          fieldLabel="Data value"
-                                                          required="{Boolean}true"/>
-                                                    </items>
-                                                </field>
-                                            </enumOptions>
+                                            <type
```

Comment:
Can we re-use the configuration from checkbox group ?

Reply:
No, we cannot use the existing configuration as the UI in checkbox group takes an array, while in checkbox we don't actually have an array, its just two 'states' ON or OFF